Tests/PythonTests/test_map.py

```
@@ -8,8 +8,20 @@
 
 import pyBaba
 
+
 def test_map_basic():
     map = pyBaba.Map(5, 5)
```

Ruff (0.15.21)

[error] Variable `map` is shadowing a Python builtin

(A001)